compiler/rustc_middle/src/middle/stability.rs

```diff
@@ -116,8 +117,12 @@ pub fn report_unstable(
         if is_soft {
             soft_handler(SOFT_UNSTABLE, span, &msg)
         } else {
-            feature_err_issue(&sess.parse_sess, feature, span, GateIssue::Library(issue), &msg)
-                .emit();
+            let mut err =
+                feature_err_issue(&sess.parse_sess, feature, span, GateIssue::Library(issue), &msg);
+            if using_vec_allocator_api {
+                err.help("consider wrapping the inner types in tuple: `<(_, _)>`");
```

Comment:
maybe this error should reconstruct the type aka `Vec<(u8,_)>` in the test below